bins/ggipcd/src/services/lifecycle/update_state.c

```diff
@@ -55,6 +59,9 @@ GglError ggl_handle_update_state(
         NULL
     );
     if (ret != GGL_ERR_OK) {
+        *ipc_error = (GglIpcError) { .error_code = GGL_IPC_ERR_SERVICE_ERROR,
+                                     .message
+                                     = GGL_STR("Received invalid paramters.") };
```

Comment:
Wouldn't this error be that the lifecycle state couldn't be updated instead of invalid parameters?

Reply:
These strings are matching GG Classic

Reply:
Good catch. Fixed.